internal/testhelpers/e2e/clean.go

```diff
@@ -79,3 +80,8 @@ func deleteUser(name string) error {
 
 	return nil
 }
+
+func CleanOutput(v string) string {
```

Comment:
This should not be needed, as we know have the `cp.ExpectLongString()` method...